lib/embed/media_tag.rb

```diff
@@ -4,6 +4,7 @@ module Embed
   # and HLS is used as a <source> within the <video> or <audio> tag.
   class MediaTag
     SUPPORTED_MEDIA_TYPES = [:audio, :video].freeze
+    MEDIA_INDEX_CONTROL_HEIGHT = 24
```

Comment:
The only things that I can think of that would allow us to pull this value into the place that it would be needed seems worse that this (add the value to the markup, then pick up the value with javascript to add to the media index markup as it is being built).